move/sources/canvas/token.move

```diff
@@ -388,39 +340,29 @@ module addr::canvas_token {
             let r = vector::pop_back(&mut rs);
             let g = vector::pop_back(&mut gs);
             let b = vector::pop_back(&mut bs);
-            draw_one(caller_addr, canvas, x, y, r, g, b);
+            draw_one(canvas_, x, y, r, g, b);
```

Comment:
i shuold be able to pass in the borrow mut canvas instead of borrowing it every pixel